CMakeLists.txt

```
@@ -19,6 +58,7 @@ endif()
 option(NEOFOAM_BUILD_EXAMPLES "Build the NeoN examples" ON)
 option(NEOFOAM_BUILD_TESTS "Build the unit tests" OFF)
 option(NEOFOAM_BUILD_BENCHMARKS "Build benchmarks" OFF)
+option(ENABLE_PROFILING "Build Score-P to enable profiling" OFF)
```

This should be either
```suggestion
option(NEOFOAM_ENABLE_PROFILING "Enable profiling support" OFF)
```
or 
```suggestion
option(NEOFOAM_WITH_SCOREP "Build with Score-P support " OFF)
```